sdk/src/main/java/io/opentelemetry/sdk/metrics/AbstractInstrument.java

```diff
@@ -156,4 +160,33 @@ final InstrumentDescriptor getInstrumentDescriptor() {
       return getMeterSharedState().getInstrumentRegistry().register(instrument);
     }
   }
+
+  static Descriptor getDefaultMetricDescriptor(
+      InstrumentDescriptor descriptor,
+      InstrumentType instrumentType,
+      InstrumentValueType instrumentValueType,
+      Aggregation aggregation) {
+    return Descriptor.create(
+        descriptor.getName(),
+        descriptor.getDescription(),
+        aggregation.getUnit(descriptor.getUnit()),
+        aggregation.getDescriptorType(instrumentType, instrumentValueType),
+        descriptor.getConstantLabels());
+  }
+
+  static Batcher getDefaultBatcher(
+      InstrumentDescriptor descriptor,
+      InstrumentType instrumentType,
+      InstrumentValueType instrumentValueType,
+      MeterProviderSharedState meterProviderSharedState,
+      MeterSharedState meterSharedState,
+      Aggregation defaultAggregation) {
```

Comment:
this method signature is getting a bit hairy. After this lands, let's think about if there might be a way to bundle some of these things together.